src/main_prog.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+
+import material_properties
+
+import generate_mesh
+
+# Allow for user input for number of materials and what kind of materials
+
+parser = argparse.ArgumentParser()
+    
+parser.add_argument("-n", "--number_materials", type=int, default=3, required=False,
+                           help="Determines how many materials are in the particle")
+
+
+parser.add_argument("-f", "--fuel", type=list(tuple), default=[('UO2',250e-6)], required=False,
```

Comment:
curiosity: how does one specify a list of tuples on the command-line??

I presume the auto-generated help message from argparse will make that clear...